AppController/lib/nginx.rb

```diff
@@ -433,6 +442,41 @@ def self.add_service_location(service_name, service_host, service_port,
     Nginx.reload
   end
 
+  def self.ensure_certs_are_in_place(project_id=nil)
+    # If the project is nil, we'll set up the self-signed certs for the
+    # internal communication.
+    target_certs = ["#{NGINX_PATH}/mycert.pem", "#{NGINX_PATH}/mykey.pem"]
+    src_certs = ["#{Djinn::APPSCALE_CONFIG_DIR}/certs/mycert.pem",
+                 "#{Djinn::APPSCALE_CONFIG_DIR}/certs/mykey.pem"]
+    ret = false
```

Comment:
I think something like `certs_modified` would be more meaningful here to indicate what this function is returning.